pimd/pim_cmd.c

```diff
@@ -9634,11 +9634,22 @@ DEFUN (no_ip_msdp_mesh_group_source,
        "mesh group local address\n")
 {
 	PIM_DECLVAR_CONTEXT(vrf, pim);
-	if (argc == 7)
-		return ip_no_msdp_mesh_group_cmd_worker(pim, vty, argv[6]->arg);
-	else
-		return ip_no_msdp_mesh_group_source_cmd_worker(pim, vty,
-							       argv[4]->arg);
+
+	return ip_no_msdp_mesh_group_source_cmd_worker(pim, vty, argv[4]->arg);
+}
+
+DEFUN (no_ip_msdp_mesh_group,
+       no_ip_msdp_mesh_group_cmd,
+       "no ip msdp mesh-group WORD",
```

Comment:
can't WORD be optional? I mean `[WORD]`.

Reply:
agreed this should be optional.